cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -63,7 +83,7 @@ <h4 class="u-mb5">
             {{ post.text | safe }}
         </p>
         {% endif %}
-        <p class="date">
+        <p class="date u-mt10 u-mb0">
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/d0382d0f984a44e81537d42624a12b5a77e23a02